### ISSUE TITLE
draw_segmentation_instances

### ISSUE THREAD

**Issue by mjack3** [pytorch/vision#6125]
### 🚀 The feature

Development a function torchvision.utils.draw_segmentation_instances to draw just the external contour instead.

### Motivation, pitch

For torchvision.utils we have draw_bounding_boxes, draw, draw_segmentation_masks, draw_keypoints but still need to do some extra preprocess for getting the instance segmentation. Would be great to have a function draw_segmentation_instances to get just the external area of the ROI.

### Alternatives

Similar to draw_bounding boxes, a optional argument fill could be added into draw_segmentation_instances to achieve this behavior.

**Comment by oke-aditya** [comment]
Can you post an example of what the output is expected? And how only contours will help?

**Comment by mjack3** [comment]
Hello!

This is what we have with the current util function

![Screenshot from 2022-06-02 15-02-54](https://user-images.githubusercontent.com/37028633/171635328-518f4956-0ea6-4334-95f2-7fc488e42b9e.png)

With this request we would have as result just the width orange border of the mask.

This can help in order to have better visualization of what we are selecting. Similar as to have filled bounding boxes or just the contour

**Comment by datumbox** [comment]
I agree that this might be useful for people who want more advanced viz options, but is TorchVision's util the right solution for them? Historically the focus of TorchVision was to offer basic utilities for people to check the results of the models. For more complex use-cases, there are better options out there. My worry is that offering such a functionality will lead to rather complex signature methods as one will need to provide not only colours for masks but also the contours.

**Comment by mjack3** [comment]
I think is  straightforward as adding a fill attribute that by default is filled.

In this way you are improving to developer don't using other libraries.

**Comment by oke-aditya** [comment]
Hi @mjack3 can you share the code that enabled you to do the same? Maybe someone looking for similar solution might open this issue and it may help them.

**Comment by mjack3** [comment]
I haven't the code. I need to develop it

**Comment by oke-aditya** [comment]
Yeah that's my doubt. How feasible and easy would it be to support this by adding `fill` param to `draw_segmentation_masks`. 

Each feature comes with a cost. The cost is maintainability and ease of use.
